components/signup/stepTwo.js

```diff
@@ -106,7 +106,7 @@ class StepTwo extends React.Component {
                                                                 false
                                                             )
                                                         }
-                                                        className="col-primary c-fw-600 p-3 cursor-pointer"
+                                                        className="col-primary c-fw-600 p-3 cursor-pointer text-underline"
```

Comment:
Rename class Name to `text-hover-underline`

Reply:
done